phial/bot.py

```diff
@@ -279,16 +279,19 @@ def send_message(self, message: Response) -> None:
             message(Response): message object to be sent to Slack
 
         '''
+        message.serialiseAttachments()
```

Comment:
See comments below regarding mutation. I feel like it would be better if this line was something more like `serialised_attachments = message.serialise_attachments()`. Or maybe even remove the function on message and just do it inline `serialised_attachments = json.dumps(message.attachments)`. 

See [this SO post](https://stackoverflow.com/a/5160278/5179209) on how json.dumps can be modified to be recursive, which we'll have to do to allow for `MessageAttachments` to have `MessageAttachmentFields`